### ISSUE TITLE
Mask head of the instance segmentation model doesn't work after export to Int8 

### ISSUE THREAD

**Issue by savanovich** [facebookresearch/d2go#135]
I'm trying to quantize the instance segmentation model (_mask_rcnn_fbnetv3a_C4_) that is trained on the custom dataset and works just fine: finds all objects on the image and outputs meaningful mask. The quantization code from the example is slightly patched to keep the same resolution as the only image in the dataset. But the final outputs have just `0.5` values in the `pred_mask`. I wonder how to say quantizer to not touch pred_mask or just how to keep mask working?

## Instructions To Reproduce the 🐛 Bug:
```python
import contextlib
import copy
import os

import cv2
import torch

from d2go.export.api import convert_and_export_predictor
from d2go.export.d2_meta_arch import patch_d2_meta_arch
from d2go.model_zoo import model_zoo
from d2go.runner import GeneralizedRCNNRunner, create_runner
from d2go.utils.testing.data_loader_helper import register_toy_coco_dataset

im_orig = cv2.imread("CDS-10132021CP1.mov_image_064.png")
height, width = im_orig.shape[:2]
new_w = 1333
new_h = 750

im = cv2.resize(im_orig, (new_w, new_h), interpolation=cv2.INTER_LINEAR)
transposed_im = im.astype("float32").transpose(2, 0, 1)
image = torch.as_tensor(transposed_im).to(torch.device("cpu"))


# ORIGINAL MODEL

CONFIG_NAME = 'mask_rcnn_fbnetv3a_C4'
WEIGHTS_PATH = f"best_{CONFIG_NAME}.pth"
confidence_threshold = 0.5


def get_config():
    runner = GeneralizedRCNNRunner()
    cfg = runner.get_default_cfg()
    cfg.merge_from_file(model_zoo.get_config_file(f"{CONFIG_NAME}.yaml"))
    cfg.MODEL_EMA.ENABLED = False
    cfg.MODEL_EMA.DEVICE = 'cpu'
    cfg.MODEL.WEIGHTS = WEIGHTS_PATH
    cfg.MODEL.ROI_HEADS.BATCH_SIZE_PER_IMAGE = 128
    cfg.MODEL.ROI_HEADS.NUM_CLASSES = 1

    cfg.MODEL.RETINANET.SCORE_THRESH_TEST = confidence_threshold
    cfg.MODEL.ROI_HEADS.SCORE_THRESH_TEST = confidence_threshold

    cfg.freeze()

    return cfg, runner


cfg, runner = get_config()
model = runner.build_model(cfg, eval_only=True)
model = model.to(torch.device("cpu"))
model.eval()

with torch.no_grad():
    inputs = [{"image": image}]
    res = model(inputs)[0]
    print(len(res['instances']))
    print(torch.min(res['instances'].pred_masks), torch.max(res['instances'].pred_masks))
    # OUTPUTS:
    # 48
    # tensor(False), tensor(True)


# QUANTIZED MODEL

patch_d2_meta_arch()


@contextlib.contextmanager
def create_fake_detection_data_loader(height, width, is_train):
    runner = create_runner("d2go.runner.GeneralizedRCNNRunner")
    cfg = runner.get_default_cfg()
    cfg.INPUT.MIN_SIZE_TRAIN = (new_h,)
    cfg.INPUT.MAX_SIZE_TRAIN = new_w
    cfg.INPUT.MIN_SIZE_TEST = new_h
    cfg.INPUT.MAX_SIZE_TEST = new_w

    with register_toy_coco_dataset("default_dataset_test", image_size=(new_w, new_h), num_images=3):
        test_loader = runner.build_detection_test_loader(
            cfg, dataset_name="default_dataset_test"
        )
        yield test_loader


with create_fake_detection_data_loader(new_h, new_w, is_train=False) as data_loader:
    predictor_path = convert_and_export_predictor(
        cfg,
        copy.deepcopy(model),
        "torchscript_int8@tracing",
        './',
        data_loader,
    )

jit_model = torch.jit.load(os.path.join(predictor_path, "model.jit"))
res = jit_model(image)
print(len(res[1]))
print(torch.min(res[2]), torch.max(res[2]))
# OUTPUTS:
# 48
# tensor(0.5000) tensor(0.5000)

```

**Comment by savanovich** [comment]
I've slightly changed the script to use real data and looks like the mask works now. The possible explanation behind this is that fake data hasn't provide observer with mask samples and they remain default values 0.5

**Comment by cloveropen** [comment]
@savanovich how to change the default values 0.5? I got the same error with mask_rcnn_fbnetv3a_dsmask_C4

**Comment by savanovich** [comment]
@cloveropen In my case change of `create_fake_detection_data_loader` to return real images helped to quantize properly

**Comment by cloveropen** [comment]
@savanovich thank U
